### PR TITLE
Add get_lock() and key_column_usage table

### PR DESCRIPTION
**Pull request by shenli**
Make Ruby ActiveRecord::Migration happy.
@coocood @zimulala 
Fix: https://github.com/pingcap/tidb/issues/1421

### CODE REVIEW

File: evaluator/builtin.go [hunk 1]
```diff
@@ -120,6 +120,12 @@ var Funcs = map[string]Func{
 	"ifnull": {builtinIfNull, 2, 2},
 	"nullif": {builtinNullIf, 2, 2},
 
+	// miscellaneous functions
+	// get_lock() and release_lock() is parsed but do nothing.
+	// It is used for preventing error in Ruby's activerecord migrations.
+	"get_lock":     {builtinLock, 2, 2},
+	"release_lock": {builtinLock, 1, 1},
```
Comment (coocood) [inline evaluator/builtin.go]: Define a 'builtinReleaseLock' is cleaner, even if it does nothing.

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**